### DISCUSSION TITLE
Hiding owned childs in tree view

### DISCUSSION THREAD

**Discussion by remyj38** [argoproj/argo-cd#20787] [Q&A]
Hi !

I manage my Rancher cluster with ArgoCD.
Everything works right, but the UI in the app view render too much items.

The cause:
A rancher cluster is a cluster.provisioning.cattle.io object. This object generate a lot of other objects, which ones own more and more objects.

So, I have a view with maybe few hundred of objects, shared on 6 levels of inheritance.

Does Argo can show only managed resources or n<sup>st</sup> level of inheritance on the app tree view ?

Thanks !

**Comment by andrii-korotkov-verkada** [comment]
I think Argo will show a whole application tree. Do you mean that essentially your app is very large and UI has problems rendering it? If so, I suggest to open an enhancement proposal.
